src/Nowruz/modules/userProfile/components/website/index.tsx

```diff
@@ -7,9 +7,9 @@ import { Link } from 'src/Nowruz/modules/general/components/link';
 import css from './website.module.scss';
 
 export interface WebsiteProps {
-  website: string;
+  websiteUrl: string;
 }
-export const Website: React.FC<WebsiteProps> = ({ website }) => {
+export const Website: React.FC<WebsiteProps> = ({ websiteUrl }) => {
```

Comment:
why changing variable name necessary  for this fix are you sure there is no other usage of this component need to fix, and better naming as it's WebsiteProps would be simply url

Reply:
@jeyem Well, I'm sure that for the usage of this component, as it is a new addition component just created two days ago. So I'm pretty sure about that. I can change it to just "url" if you suggest.